creator-node/src/routes/tracks.js

```diff
@@ -421,6 +421,22 @@ router.post(
           throw new Error('Cannot create track without transcodedTrackUUID.')
         }
 
+        // Verify that blockchain track id is owned by user attempting to upload it
+        const serviceRegistry = req.app.get('serviceRegistry')
+        const { libs } = serviceRegistry
+        const trackResp = await libs.contracts.TrackFactoryClient.getTrack(
+          blockchainTrackId
+        )
+        if (
+          !trackResp?.trackOwnerId ||
+          // eslint-disable-next-line eqeqeq
+          trackResp.trackOwnerId != req.session.userId
```

Comment:
i think we should parse this into an int and do a `!==` like

```suggestion
          parseInt(trackResp.trackOwnerId, 10) !== parseInt(req.session.userId, 10)
```

Reply:
done 👍